apps/condo/domains/common/components/GraphQlSearchInput.tsx

```diff
@@ -63,17 +64,32 @@ export const GraphQlSearchInput: React.FC<ISearchInputProps> = (props) => {
         getInitialValueQuery,
         infinityScroll,
         disabled,
+        placeholder,
+        value: selectedValues,
         eventName: propEventName,
         eventProperties = {},
         ...restProps
     } = props
+
+    const intl = useIntl()
+    const LoadingMessage = intl.formatMessage({ id: 'Loading' })
+
     const client = useApolloClient()
     const [selected, setSelected] = useState('')
     const [isLoading, setLoading] = useState(false)
     const [data, setData] = useState([])
     const [isAllDataLoaded, setIsAllDataLoaded] = useState(false)
     const [value, setValue] = useState('')
 
+    const _placeholder = useMemo(() => (isLoading && !data.length) ? LoadingMessage : placeholder,
+        [LoadingMessage, data.length, isLoading, placeholder])
+
+    const _selectedValue = useMemo(() => (isLoading && !data.length) ? [] : selectedValues,
```

Comment:
you're calculating `isLoading && !data.length` several times, it's code duplication. better to calculate it once, store to a variable and reuse it.